lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -55,6 +55,22 @@ def common_attributes?(selector)
           attributes(selector).keys.difference(COMMON_OPTIONS).none?
         end
 
+        # @param selector [String]
+        # @return [Array<String>]
+        # @example
+        #   pseudo_classes('button:not([disabled])') # => ['not()']
+        #   pseudo_classes('a:enabled:not([valid])') # => ['enabled', 'not()']
+        def pseudo_classes(selector)
+          # Attributes must be excluded or else the colon in the `href`s URL
+          # will also be picked up as pseudo classes.
+          # "a:not([href='http://example.com']):enabled" => "a:not():enabled"
+          ignored_attribute = selector.gsub(/\[.*?\]/, '')
+          # "a:not():enabled" => [["not():enabled"]]
+          pseudo_class = ignored_attribute.scan(/:(.*)/)
+          # [["not():enabled"]] => ["not()", "enabled"]
+          pseudo_class.flatten.join.split(':')
```

Comment:
```suggestion
          # "a:not():enabled" => ["not()", "enabled"]
          ignored_attribute.scan(/:([^:]*)/).flatten
```

Reply:
It looks very good. Thank you very much.